src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveSummaryMasteredResourceTest.kt

```diff
@@ -136,15 +140,15 @@ class IncentiveSummaryMasteredResourceTest : SqsIntegrationTestBase() {
     )
   }
 
-  suspend fun prisonerIepLevel(
+  private suspend fun prisonerIepLevel(
     bookingId: Long,
     prisonerNumber: String,
     level: String = "STD",
     reviewType: ReviewType = ReviewType.REVIEW,
     reviewTime: LocalDateTime = now(),
     current: Boolean = false,
     prisonId: String = "MDI"
-  ) = repository.save(
+  ) = prisonerIepLevelRepository.save(
```

Comment:
These entities used to remain after test case completion, restricting incentive level tests.